requirements-dev.txt

```diff
@@ -91,7 +91,7 @@ pydantic-core==2.6.3; python_version >= '3.7'
 pynacl==1.5.0
 python-dateutil==2.8.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 redis==5.0.0; python_version >= '3.7'
-securesystemslib[crypto,pynacl]==0.28.0; python_version ~= '3.7'
+securesystemslib[crypto,pynacl]==0.29.0; python_version ~= '3.7'
```

Comment:
Do not merge!

See:
- https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/372
- https://github.com/repository-service-tuf/repository-service-tuf/actions/runs/6113062461/job/16591789595?pr=456